devices.js

```diff
@@ -14205,6 +14205,46 @@ const devices = [
             await reporting.instantaneousDemand(endpoint);
         },
     },
+    {
+        zigbeeModel: ['1GBatteryDimmer50AU'],
+        model: 'AU-A1ZBR1GW',
+        vendor: 'Aurora Lighting',
+        description: 'AOne one gang wireless battery rotary dimmer',
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_step_color_temperature],
+        toZigbee: [tz.light_brightness_step],
```

Comment:
Why was `tz.light_brightness_step` added here? Would expect it to be empty. Same for https://github.com/Koenkk/zigbee-herdsman-converters/pull/2032/files#diff-5520dd49755e0a353d3857b8963a91cb4b8d87a01b6b8eb7bb6cce1c180eac83R14233

Reply:
sorry not a pro got confused by the controller giving out the step value:

DEBUG: Received Zigbee message from 'Single Rotary Dimmer Remote', type 'commandStep', cluster 'genLevelCtrl', data '{"stepmode":0,"stepsize":26,"transtime":3}' from endpoint 1 with groupID 0
INFO: MQTT publish: topic 'zigbee2mqtt/Single Rotary Dimmer Remote', payload '{"action":"brightness_step_up","action_step_size":26,"action_transition_time":0.03,"linkquality":60,"update":{"state":"idle"},"update_available":false}'

Reply:
thought passing on the value might be of some importance

Reply:
passing = fromZigbee, sending to device = toZigbee